source/Cosmos.System2/Graphics/Fonts/ASC16.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Cosmos.System.Graphics;
+using System;
+using System.Drawing;
+using System.IO;
+using System.Text;
+
+namespace Cosmos.System.Graphics.Fonts
+{
+    public static class ASC16
```

Comment:
This class should be removed for now. We need to enhance the font system to support multiple font formats